src/PKSim.UI/BootStrapping/PKSimApplication.cs

```diff
@@ -64,5 +67,30 @@ protected override void OnRun()
             throw;
          }
       }
+
+      public bool PreFilterMessage(ref Message m)
+      {
+         if (!isWindowsKeyDownMessage(m))
+            return false;
+
+         switch (keyCodeFrom(m))
+         {
+            case (int)(Keys.Control | Keys.W):
+               _mainViewPresenter.ActivePresenter?.Close();
```

Comment:
No side effects here? 
(e.g. occasionally closing tabs which are supposed to be always visible - like "Parameters" in a simulation view etc.)

Reply:
that should be the tabbed MDI presenter only.

Reply:
Like closing with the close context menu, except you can activate it when the document is not active... like, you've selected a node in the building block explorer

![image](https://github.com/user-attachments/assets/a397ddfc-e7bd-436e-9cdb-0a3423b673d8)

Reply:
ok